charts/mungegithub/chart/values.kubernetes.contrib.yaml

```diff
@@ -3,9 +3,7 @@ config:
   http-cache-dir: /cache/httpcache
   organization: kubernetes
   project: contrib
-  # Make sure approval-handler and blunderbuss run before submit-queue.
-  # Otherwise it's going to take an extra-cycle to detect the label change.
-  pr-mungers: approval-handler,blunderbuss,lgtm-after-commit,submit-queue,needs-rebase
+  pr-mungers: blunderbuss,lgtm-after-commit,submit-queue,needs-rebase
```

Comment:
this is out of date relative to the actual configmap being used... maybe these should be removed? for now my goal was just to remove the approval-handler reference